modules/custom/social_auth_extra/social_auth_extra.module

```diff
@@ -197,9 +202,11 @@ function social_auth_extra_form_user_register_form_submit($form, FormStateInterf
         '@community_name' => \Drupal::config('system.site')->get('name'),
       ]));
 
-      $form_state->setRedirect('entity.user.edit_form', [
-        'user' => $account->id(),
-      ]);
+      // Redirect to profile.
```

Comment:
@frankgraave I'm still discussing with Taras to move this to social_profile or social_user instead since the idea is to make this a separate module at one point. I'll make the change tomorrow or later today!